javatests/build/bazel/tests/integration/BazelBaseTestCaseTest.java

```diff
@@ -72,6 +73,17 @@ public void scratchFileShouldCreateFileAndWorkspaceContentsContainThatFile() thr
     org.hamcrest.MatcherAssert.assertThat(actualScratchFileContent, is(optionalWithValue(equalTo(content))));
   }
 
+  @Test
+  public void newWorkspaceCreatesANewWorkspace() throws IOException {
+      String path = "somePathForNewWorkspace";
```

Comment:
these two lines are probably redundant right?

Reply:
but you're not creating it anywhere.
Also maybe rename the test to say `...CreatesANewCleanWorkspace`?

Reply:
Renamed the test.
WDYM not creating it?
1. scratch file
2. create a new workspace
3. list files and make sure the file isn't there